cmd/operator-sdk/generate/bundle/bundle.go

```diff
@@ -69,6 +76,10 @@ func (c bundleCmd) runKustomize(cfg *config.Config) error {
 		return fmt.Errorf("error generating ClusterServiceVersion: %v", err)
 	}
 
+	if err := kustomize.WriteIfNotExist(c.outputDir, kubebuilderKustomization); err != nil {
+		return err
+	}
+
 	if !c.quiet {
 		fmt.Println("Bases generated successfully in", c.outputDir)
 	}
```

Comment:
It shows not be working. See: (I just follow up the quickstart steps and executed `make bundle`)

```
$ make bundle
/Users/camilamacedo/go/bin/controller-gen "crd:trivialVersions=true" rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
operator-sdk generate bundle -q --kustomize
kustomize build config/bundle | operator-sdk generate bundle -q --manifests --version 0.0.1
Error: accumulating resources: couldn't make target for path '/Users/camilamacedo/go/src/test-make/config/samples': unable to find one of 'kustomization.yaml', 'kustomization.yml' or 'Kustomization' in directory '/Users/camilamacedo/go/src/test-make/config/samples'
```

<details>
<summary>
see here all steps`
</summary>

```
camilamacedo@Camilas-MacBook-Pro ~/go/src $ cd test-make/
camilamacedo@Camilas-MacBook-Pro ~/go/src/test-make $ operator-sdk init --domain my.domain
Writing scaffold for you to edit...
Get controller runtime:
$ go get sigs.k8s.io/controller-runtime@v0.6.0
Update go.mod:
$ go mod tidy
Running make:
$ make
/Users/camilamacedo/go/bin/controller-gen object:headerFile="hack/boilerplate.go.txt" paths="./..."
go fmt ./...
go vet ./...
go build -o bin/manager main.go
Next: define a resource with:
$ operator-sdk create api
camilamacedo@Camilas-MacBook-Pro ~/go/src/test-make $ kubebuilder create api --group webapp --version v1 --kind Guestbook
Create Resource [y/n]
y
Create Controller [y/n]
y
Writing scaffold for you to edit...
api/v1/guestbook_types.go
controllers/guestbook_controller.go
Running make:
$ make
/Users/camilamacedo/go/bin/controller-gen object:headerFile="hack/boilerplate.go.txt" paths="./..."
go fmt ./...
go vet ./...
go build -o bin/manager main.go
camilamacedo@Camilas-MacBook-Pro ~/go/src/test-make $ make install
/Users/camilamacedo/go/bin/controller-gen "crd:trivialVersions=true" rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
operator-sdk generate bundle -q --kustomize

Display name for the operator (required): 
> test

Description for the operator (required): 
> test

Provider's name for the operator (required): 
> test

Any relevant URL for the provider name (optional): 
> 

Comma-separated list of keywords for your operator (required): 
> 
No list provided. 
Comma-separated list of keywords for your operator (required): 
> 
No list provided. 
Comma-separated list of keywords for your operator (required): 
> 
No list provided. 
Comma-separated list of keywords for your operator (required): 
> 
No list provided. 
Comma-separated list of keywords for your operator (required): 
> test,test

Comma-separated list of maintainers and their emails (e.g. 'name1:email1, name2:email2') (required): 
> name
,
/Users/camilamacedo/go/bin/kustomize build config/crd | kubectl apply -f -
customresourcedefinition.apiextensions.k8s.io/guestbooks.webapp.my.domain created
camilamacedo@Camilas-MacBook-Pro ~/go/src/test-make $ ,
-bash: ,: command not found
camilamacedo@Camilas-MacBook-Pro ~/go/src/test-make $ make bundle
/Users/camilamacedo/go/bin/controller-gen "crd:trivialVersions=true" rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
operator-sdk generate bundle -q --kustomize
kustomize build config/bundle | operator-sdk generate bundle -q --manifests --version 0.0.1
Error: accumulating resources: couldn't make target for path '/Users/camilamacedo/go/src/test-make/config/samples': unable to find one of 'kustomization.yaml', 'kustomization.yml' or 'Kustomization' in directory '/Users/camilamacedo/go/src/test-make/config/samples'
```